autogpt_platform/backend/backend/api/features/platform_linking/routes_test.py

```
@@ -0,0 +1,891 @@
+@asynccontextmanager
+async def _fake_transaction():
+    """Stub for backend.data.db.transaction used in route tests.
+
+    The real transaction() opens a Prisma tx which binds asyncio primitives to
+    the running event loop. Tests run in their own loops via pytest-asyncio, so
+    we swap this in to avoid cross-loop errors.
+    """
+    yield MagicMock()
+
+
+from backend.api.features.platform_linking.auth import check_bot_api_key
```

Ruff (0.15.10)

[error] Module level import not at top of file

(E402)